Monitoring in a Box/Ansible/group_vars/all.yml

```diff
@@ -30,26 +17,10 @@ CreateFullClone: --full true
 # sgpools
 TargetStorage: local
 
-## Which image format to use
-# qcow2
-# vmdk
-# raw
-ImageFormat: vmdk
-
-## New account username
-NewVmUsername: mib
-
-## New account password
-NewVmPassword: [key in a password]
-
 ## VM's Vlan Tag
 # Change the number after the "=" sign to the desired Vlan number, Comment away if Vlan is not required
 VlanTag: #",tag=88"
 
-## VM's IP address
-NewVmIP:
-  - 10.0.1.240
-
 ### Network gateway ###
 GwIP: 10.0.1.5
```

Comment:
Gateway IP and DNS search domain should be a hostvar. Every environment is different. Actually DNS search domain is unnecessary so you can omit it.

Reply:
But this script is specific to our environment , for example the pve node to create on and also the template names. Which is why I put the gateway IP in the group_vars , since its already dependent on the environment.

Reply:
Environment-related vars is specific to the environment so it should not be a global var in `all.yml`. The rationale for putting it in hostvars is to have 1x hostvar per deployment.

Reply:
This gateway ip is actually part of the set up environment in the test cluster, so no matter which host, they will be using this gateway ip for internet access during the setup. Also I worried that if i put it in the host_var, the user might accidentally edit it. Which is why I put it in all.yml, out of sight from the user.

Reply:
I think there's a misunderstanding. The Ansible scripts will not be run by the user so only you guys should be using it. Is there something else I'm missing?

Reply:
Yeah this is so we can create and set up a MiB VM. After which we will just export the VM image and give the user. Then we will advice user to access the /etc/netplan config to set up the network to suit their environment.

Reply:
Correct lor. That means the vars can be in hostvars.

Reply:
is it? even tho all the VMs being set up will use the same gateway ip?

Reply:
Ok sorry I understand your point now. Rename the variables to better describe what's going on and comment the code as well. That way there's lesser confusion. Thanks!